index.html

```diff
@@ -109,78 +113,83 @@ <h4 class="project-client">Canopy</h4>
             <img src="./image/Counter.svg" alt="counter" />
             <span class="year">2015</span>
           </nav>
           <p class="project-description">
-            A daily selection of privately personalized reads; no accounts or
-            sign-ups required.
+            Exploring the future of media in 
+            Facebook's first Virtual Reality app; a place to discover and enjoy 360 photos and videos on Gear VR.
           </p>
           <ul class="pro-tool">
             <li class="tool">html</li>
+            <li class="tool">Ruby on rails</li>
             <li class="tool">css</li>
             <li class="tool">javascript</li>
           </ul>
-          <button class="more">see project</button>
+          <button class=" more more-about">see project</button>
         </div>
       </div>
       <div class="project multipost2-project">
-        <img
-          src="./image/Snapshoot Portfolio4.svg "
-          alt="tonic"
-          class="project-img"
-        />
-
         <div class="project-details">
           <h2 class="project-title">
-            Multipost <br />
-            stories
+            Uber Navigation
           </h2>
           <nav class="project-nav">
-            <h4 class="project-client">Canopy</h4>
+            <h4 class="project-client">Uber</h4>
             <img src="./image/Counter.svg" alt="counter" />
-            <span class="pro-lang">Back End Dev</span>
+            <span class="pro-lang">Lead Developer</span>
             <img src="./image/Counter.svg" alt="counter" />
-            <span class="year">2015</span>
+            <span class="year">2018</span>
           </nav>
           <p class="project-description">
-            A daily selection of privately personalized reads; no accounts or
-            sign-ups required.
+            A smart assistant to make driving more safe, efficient, and fun by 
+            unlocking your most expensive computer: your car.
           </p>
           <ul class="pro-tool">
             <li class="tool">html</li>
+            <li class="tool">Ruby on rails</li>
             <li class="tool">css</li>
             <li class="tool">javascript</li>
           </ul>
           <button class="more">see project</button>
         </div>
+        <img
+        src="image\Snapshoot Portfolio 21.png "
+        alt="tonic"
+        class="project-img"
+      />
       </div>
     </section>
     <section class="about-myself">
       <div class="myself">
-        <h2 class="name">About<br />Myself</h2>
+       <div class="myself-intro"> 
+        <h2 class="name">About<br/>Myself</h2>
         <p class="text">
           Hello I’m a software developer! I can help you build a product ,
           feature or website Look through some of my work and experience! If you
           like what you see and have a project you need coded, don’t hestiate to
           contact me.
         </p>
 
-        <div class="contact">
+        
```

Comment:
- [ ] You should stick to the design as much as possible. please fix the spacing as shown in the figure below and also use the same color for `hx` tag also fix the spacing from left and right.
- [ ] Also the image for javascript is missing please fix it.
### Your design
![his](https://user-images.githubusercontent.com/81727988/213680441-6f2d935d-44eb-4b83-8f0f-fcf5c90e0dfe.jpg)

### Figma design
![fimga](https://user-images.githubusercontent.com/81727988/213680472-a4d3d814-76fa-470f-8270-a70f51b90219.jpg)